lib/gems/pending/util/miq-xml.rb

```diff
@@ -41,7 +41,7 @@ def self.xml_document(xmlClass)
       when :xmlhash
```

Comment:
tangent: do we have enough coverage to know if we can drop `:xmlhash` or `:rexml`?

Reply:
I had assumed we only used this in a few rare places, but besides smart state (and specs), looks like we use it in kubernetes, vmware, ui classic (only in 1-2 files each) along with a dozen core models.